cypress/e2e/po/pages/extensions.po.ts

```diff
@@ -0,0 +1,249 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+
+export default class ExtensionsPo extends PagePo {
+  static url: string = '/c/local/uiplugins'
+  static goTo(): Cypress.Chainable<Cypress.AUTWindow> {
+    return super.goTo(ExtensionsPo.url);
+  }
+
+  constructor() {
+    super(ExtensionsPo.url);
+  }
+
+  // screen title
+  title(): Cypress.Chainable<string> {
+    return this.self().getId('extensions-page-title').invoke('text');
+  }
+
+  // install extensions operator
+  installExtensionsOperatorIfNeeded(attempt = 0) {
+    // this will make sure we wait for the page to render first content
+    // so that the attemps aren't on a blank page
+    this.title();
+
+    if (attempt > 20) {
+      return;
+    }
+    if (Cypress.$('[data-testid="extension-enable-operator"]').length > 0) {
```

Comment:
the `.is(':visible')` does not work as the `is` is not chainable apparently.

Doing conditional testing is quite a problem in e2e tests (at least without installing any more packages). If you want to take a stab at this, please do, otherwise let's move this forward if you don't mind, as this PR has been open for several weeks now.